apps/users/utils.py

```diff
@@ -9,4 +9,4 @@ def set_session_language(user_email, language=None, request=None):
         language = User.objects.get(email=user_email).language
     activate(language)
     if hasattr(request, "session"):
-        request.session[LANGUAGE_SESSION_KEY] = language
+        request.session[settings.LANGUAGE_COOKIE_NAME] = language
```

Comment:
does this still do what we want it to do? django deprecated storing the language in the session and uses a cookie, so this code doesn't affect any django internals and I can't find any code from us reading the value.

Reply:
it is imported in apps/accounts/views.py
but you are right, cookie should be used on the browser. will remove the request part then